server/models/services/GithubWorkflowService.js

```diff
@@ -0,0 +1,157 @@
+//import token from '../.github-app-token.json';
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+const path = require('path');
+
+const ONE_MINUTE = 60;
+
+// Assigned by GitHub
+const GITHUB_APP_ID = '395709';
+
+// > your JWT must be signed using the RS256 algorithm.
+//
+// https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app
+const ALGORITHM = 'RS256';
+
+// > Note: [The `workflow_dispatch event] will only trigger a workflow run if
+// > the workflow file is on the default branch.
+//
+// https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+const WORKFLOW_FILE_NAME = 'nvda-chrome.yml';
+const WORKFLOW_REPO = 'bocoup/aria-at-gh-actions-helper';
+
+// Generated from the GitHub.com UI
+let privateKey = null;
+try {
+    privateKey = fs.readFileSync(
+        path.join(__dirname, '../../../jwt-signing-key.pem')
+    );
+} catch (e) {
+    // Empty catch block, if the key is null we will hear about it later.
+}
```

Comment:
In production scenarios, multiple days may pass between the deployment of this code and the use of the feature. If the person deploying has failed to provide this critical file, we want them to be made aware of that immediately.

Reply:
My plan: re-throw on production/ENABLE_AUTOMATION only - ignore the error about missing file in dev environments that aren't using it to create github jobs?

Reply:
I'd like to shrink this testing seam (really, *all* testing seems) as much as possible. Doing so makes it easier to reason about each component's behavior. In this case, since the project uses `require`, we can allow this exception to propagate implicitly if we defer the loading of the `GithubWorkflowService.js` module in to the existing seam in `CollectionJobService.js`.